common/api/frontend-tiles.api.md

```diff
@@ -80,27 +80,18 @@ export type GraphicRepresentation = {
     format: GraphicRepresentationFormat;
     dataSource: DataSource;
 } & ({
-    status: Omit<GraphicRepresentationStatus, GraphicRepresentationStatus.Complete>;
+    status: Exclude<GraphicRepresentationStatus, "Complete">;
     url?: string;
 } | {
-    status: GraphicRepresentationStatus.Complete;
+    status: "Complete";
     url: string;
 });
 
 // @beta
-export type GraphicRepresentationFormat = "IMDL" | "3DTILES" | string;
+export type GraphicRepresentationFormat = "3DFT" | "3DTiles" | "CESIUM" | "IMODEL" | string;
```

Comment:
Do we need type "string" here?

Reply:
Please refer to [previous comments](https://github.com/iTwin/itwinjs-core/pull/7570#discussion_r1927383203).

Reply:
Makes sense.